tabular/src/autogluon/tabular/models/vowpalwabbit/vowpalwabbit_model.py

```diff
@@ -94,45 +76,35 @@ def _fit(self,
         final_training_data = y + X_series
         final_training_data = final_training_data.tolist()
 
+        extra_params = {
+            'cache_file': 'train.cache',
+            'holdout_off': True,
+            'quiet': True,
+        }
+
         # Initialize the model
         if self.problem_type in PROBLEM_TYPES_CLASSIFICATION:
             # Ref: https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Predicting-probabilities#multi-class---oaa
-            self.labels = list(set(y.tolist()))
-            self.oaa: int = len(self.labels)
-            print(f'Setting oaa to {self.oaa}')
-            self.model = pyvw.vw(
-                **params,
-                oaa=self.oaa,
-                cache_file='train.cache',
-                holdout_off=True,
-                probabilities=True,  # Only to be used for classification
-                quiet=True,
-            )
-        elif self.problem_type in PROBLEM_TYPES_REGRESSION:
-            self.model = pyvw.vw(
-                **params,
-                cache_file='train.cache',
-                holdout_off=True,
-                quiet=True,
-            )
+            extra_params['oaa'] = len(set(y.tolist()))
+            print(f'Setting oaa to {extra_params["oaa"]}')
+            extra_params['probabilities'] = True
+        self.model = pyvw.vw(**params, **extra_params)
+
         # Train the model
-        for epoch in range(1, epochs + 1):
+        np.random.seed(42)
```

Comment:
I know its fun to use 42, but for consistency with all models in AutoGluon, we try to use seed 0 by default. May also be better to define `seed=0` near the top of fit in case we want to make it configurable in future.

Reply:
Addressed in killerSwitch#1